### ISSUE TITLE
Feature request: Generate moves in batch

### ISSUE THREAD

**Issue by liverex** [yishn/LeelaSabaki#5]
Can we open a sgf file, and then let LeelaSabaki generate variations for specified moves?

**Comment by yishn** [comment]
You can navigate to a specific position and ask LeelaSabaki to generate a move. (<kbd>F5</kbd>)

**Comment by liverex** [comment]
Yes, I know this feature. I even implemented a python script controlling LeelaSabaki to generate some moves as specified. However, it would be very useful if it is supported by LeelaSabaki natively.

**Comment by yishn** [comment]
So I'm not understanding what you propose... What you mean by 'for specified moves'? How do you want to specify the moves?
